### PR TITLE
Rgi conditional mv

### PR DESCRIPTION
**Pull request by jfy133**
To address https://github.com/nf-core/funcscan/issues/376

I'm aware the repeated `if` statements are not the most efficient, but my `bash` wasn't strong enough to get a working loop with all the escapes etc...

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by itrujnara — APPROVED** [review]
Not the prettiest code ever, but I see no better solution to this one

File: modules/nf-core/rgi/main/main.nf [hunk 1]
```diff
@@ -60,7 +60,16 @@ process RGI_MAIN {
         --input_sequence $fasta
 
     mkdir temp/
-    mv *.xml *.fsa *.{nhr,nin,nsq} *.draft *.potentialGenes *{variant,rrna,protein,predictedGenes,overexpression,homolog}.json temp/
+    mv *.xml *.fsa *.{nhr,nin,nsq} *.draft *.potentialGenes temp/
+
+    ## These files not always generated, so only move if they exist
+    ## Taken from https://stackoverflow.com/a/54655320 (RIP) by @TSJNachos117
+    if [ "\$(ls -A | grep -i \\variant.json\\\$)" ]; then mv *variant.json temp/; fi
```
Comment (SPPearce) [inline modules/nf-core/rgi/main/main.nf:67]: It'll do provided you align the semi-colons.

**Comment by jfy133** [comment]
Thank you @itrujnara

File: modules/nf-core/rgi/main/main.nf (hunk 1 above)
Comment (jfy133) [reply modules/nf-core/rgi/main/main.nf]: done. Feels weird doing harshil align on bash but first time for anything

**Pull request merged**